bin/pygrb/pycbc_grb_trig_combiner

```diff
@@ -0,0 +1,407 @@
+                            )
+                            bar.update()
+            bar.close()
+            if verbose:
+                print("{} written to {}".format(bin_, outf))
+
+
+def read_segment_files(segdir):
+    segs = {}
+    for name, filename in {
+            "buffer": "bufferSeg.txt",
+            "off": "offSourceSeg.txt",
+            "on": "onSourceSeg.txt",
+    }.items():
+        try:
+            with open(os.path.join(segdir, filename), "r") as f:
+                segs[name], = fromsegwizard(f)
+        except ValueError as exc:
+            exc.args = ("more than one segment, an error has occured",)
+            raise
+    return segs
+
+
+# -- parse command line -------------------------
+
+parser = argparse.ArgumentParser(
+    description=__doc__,
+)
+
+parser.add_argument(
+    "-v",
+    "--verbose",
+    action="store_true",
+    default=False,
+    help="verbose output with microsecond timer (default: %(default)s)",
+)
+parser.add_argument(
+    "-V",
+    "--version",
+    action="version",
+    version=__version__,
+    help="show version number and exit",
+)
+
+# tags
+parser.add_argument(
+    "-i",
+    "--ifo-tag",
+    required=True,
+    help="the IFO tag, e.g. H1L1",
+)
+parser.add_argument(
+    "-u",
+    "--user-tag",
+    default="PYGRB",
+    type=str.upper,
+    help="the user tag (default: %(default)s)",
+)
+parser.add_argument(
+    "-j",
+    "--job-tag",
+    type=str.upper,
+    help="the job tag, for use when more than one trig_combiner "
+         "job is included in a workflow",
+)
+parser.add_argument(
+    "-S",
+    "--slide-tag",
+    type=str.upper,
+    help="the slide tag, used to differentiate long slides",
+)
+
+# run parameters
+parser.add_argument(
+    "-n", "--grb-name",
+    type=str.upper,
+    help="GRB event name, e.g. 010203",
+)
+parser.add_argument(
+    "-T",
+    "--num-trials",
+    type=int,
+    default=6,
+    help="The number of off source trials, default: %(default)d",
+)
+parser.add_argument(
+    "-p",
+    "--trig-start-time",
+    type=int,
+    required=True,
+    help="The start time of the analysis segment",
+)
+parser.add_argument(
+    "-a",
+    "--segment-dir",
+    required=True,
+    help="directory holding buffer, on and off source segment files",
+)
+parser.add_argument(
+    "-s",
+    "--short-slides",
+    action="store_true",
+    help="Did analysis use short time slides?",
+)
+parser.add_argument(
+    "-t",
+    "--long-slides",
+    action="store_true",
+    help="Are these triggers from long time slides?",
+)
+
+# input/output
+parser.add_argument(
+    "-o",
+    "--output-dir",
+    default=os.getcwd(),
+    help="output directory (default: %(default)s)",
+)
+parser.add_argument(
+    "-f",
+    "--input-files",
+    nargs="*",
+    required=True,
+    metavar="TRIGGER FILE",
+    help="read in listed trigger files",
+)
+
+args = parser.parse_args()
+
+vprint = print if args.verbose else str
+
+vprint("-- Hello")
```

Comment:
Don't think we need 'hello'

Reply:
There's nothing wrong with a polite welcome...